zuul.d/edpm_periodic.yaml

```diff
@@ -66,3 +66,20 @@
       cifmw_repo_setup_branch: antelope
       openstack_release: "{{ cifmw_repo_setup_branch }}"
       cifmw_edpm_build_images_push_registry_namespace: "podified-{{ cifmw_repo_setup_branch }}-centos9"
+
+# Whitebox Neutron tempest plugin job
+- job:
```

Comment:
Could you please add a `description:` @raukadah

Reply:
Done!